src/detectors/Java/JavaComponentDetector.ts

```diff
@@ -12,7 +12,7 @@ export class JavaComponentDetector implements IProjectComponentDetector {
   }
 
   async detectComponent(langAtPath: LanguageAtPath): Promise<ProjectComponent[]> {
-    const hasSomeAndroidPackage = this.packageInspector.hasPackage(new RegExp('com.android.support:*'));
+    const hasSomeAndroidPackage = this.packageInspector.hasPackage(new RegExp('com.android.*'));
```

Comment:
I've checked this with my friend, a senior Android Developer, and he said it's not possible to check whether it's an Android project like this since every developer can name the package how they want to. But he said it's possible to detect whether it's an Android project by checking the presence of the `AndroidManifest.xml` file in the project.

Reply:
It makes sense, I will make changes